base/utils/R/utils.R

```diff
@@ -207,9 +207,9 @@ zero.bounded.density <- function(x, bw = "SJ", n = 1001) {
 summarize.result <- function(result) {
```

Comment:
```suggestion
##' @importFrom rlang .data
summarize.result <- function(result) {
```

Reply:
I guess you'll need to add rlang to the Imports section of DESCRIPTION too. I'd do it here, but I don't think GitHub's suggest changes feature works on files not already changed in the PR.

Reply:
@infotroph I think you can suggest changes on the source branch, e.g. here: https://github.com/jessicaguo/pecan/edit/track-RE/base/utils/DESCRIPTION

I can do it but if this is the right fix, is this something that we will have to do throughout a lot of packages with similar dplyr syntax (it wasn't clear from the links I found)?

Reply:
> you could suggest changes on the source branch

Cool! Thank you.

> is this something that we will have to do throughout

Eventually, yes. It's definitely annoying to write, but IMO not any more so than the various other workarounds (e.g. `globalVariables`), and is the approach recommended by the Tidyverse developers. 

When should "eventually" be? It's an easy but tedious fix for a problem that's mostly just annoying (it could be masking bugs, but in practice we'd probably notice them some other way). So I'd say "as time allows, probably one package at a time whenever reminded by a PR like this one." By my count we're currently ignoring 571 `no visible binding for global variable` NOTEs across all of PEcAn, and my wild guess is that half to two-thirds of these are Tidyverse calls.